.github/workflows/cut-rc.yml

```diff
@@ -244,22 +242,46 @@ jobs:
             echo ":arrow_right: Close the newly created milestone [${{needs.variables.outputs.next-milestone}}](${{github.event.milestone.html_url}}) when ready." >> $GITHUB_STEP_SUMMARY
           fi
 
-  pre-release:
-    name: Publish a GitHub pre-release
-    needs: [variables, cut-rc]
-    runs-on: ubuntu-latest
-    steps:
+      # - name: Run Gosec security scanner
+      #   uses: securego/gosec@master
+      #   with:
+      #     args: "-no-fail -out ${{ steps.tag.outputs.report-file-name }} ./..."
+      # - run: |
+      #     cat <<EOF >> $GITHUB_STEP_SUMMARY
+      #     # gosec Report
+
+      #     \`\`\`
+      #     EOF
+      #     cat "$GITHUB_WORKSPACE/${{ steps.tag.outputs.report-file-name }}" >> $GITHUB_STEP_SUMMARY
+      #     echo '```' >> $GITHUB_STEP_SUMMARY
+
```

Comment:
Maybe let's remove it? 
Regardless of it being a comment, it feels weird that this is located in `Create next milestone`

Reply:
I think it's a separate job `Run Gosec security scanner`, just looks to be in the previous job because of the comment formatting.
Removed all references in workflows to Gosec